conda_build/build.py

```diff
@@ -2868,32 +2868,6 @@ def warn_on_use_of_SRC_DIR(metadata):
             )
 
 
-@deprecated(
-    "3.16.0",
-    "24.7.0",
-    addendum=(
-        "Test built packages instead, not recipes "
-        "(e.g., `conda build --test package` instead of `conda build --test recipe/`)."
-    ),
-    deprecation_type=FutureWarning,  # we need to warn users, not developers
-)
```

Comment:
Oops! This was actually just recently marked as deprecated (see https://github.com/conda/conda-build/pull/5352). We need to change the removal version to 24.9 instead!!